jmx-metrics/src/integrationTest/java/io/opentelemetry/contrib/jmxmetrics/target_systems/WildflyIntegrationTest.java

```diff
@@ -0,0 +1,128 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.contrib.jmxmetrics.target_systems;
+
+import static org.assertj.core.api.Assertions.entry;
+
+import io.opentelemetry.contrib.jmxmetrics.AbstractIntegrationTest;
+import java.time.Duration;
+import org.junit.jupiter.api.Test;
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.containers.Network;
+import org.testcontainers.containers.wait.strategy.Wait;
+import org.testcontainers.utility.MountableFile;
+
+class WildflyIntegrationTest extends AbstractIntegrationTest {
+
+  WildflyIntegrationTest() {
+    super(/* configFromStdin= */ false, "target-systems/wildfly.properties");
+  }
+
+  /* In order to create a JMX connection to WildFLy, the scraper requires an additional
+  client jar in the classpath. To facilitate this, the scraper runs in the same container,
+  which gives it access to the client jar packaged with WildFly. */
+  @Override
+  protected GenericContainer<?> buildScraper(String otlpEndpoint) {
+    String scraperJarPath = System.getProperty("shadow.jar.path");
+
+    return new GenericContainer<>("jboss/wildfly:23.0.1.Final")
+        .withNetwork(Network.SHARED)
+        .withNetworkAliases("wildfly")
+        .withCopyFileToContainer(
+            MountableFile.forHostPath(scraperJarPath), "/app/OpenTelemetryJava.jar")
```

Comment:
nit 
```suggestion
            MountableFile.forHostPath(scraperJarPath), "/app/OpenTelemetryJMXMetrics.jar")
```

Reply:
I've updated the jar name. However, I pulled that name originally from AbstractIntegrationTest, so we might want to change it there as well at a later point.